Gemfile

```diff
@@ -26,6 +26,7 @@ gem 'faraday_middleware'
 gem 'httpclient'
 gem 'attr_encrypted'
 gem 'olive_branch'
+gem 'sidekiq'
```

Comment:
Any desire to lock these down to minor versions to prevent undesired upgrade issues?